src/tests/VisibilityChanger.test.js

```diff
@@ -0,0 +1,172 @@
+            if (id !== 'LinkedIn'){
+                wordsArry = id.match(/[A-Z][a-z]+/g);
+            } else {
+                wordsArry = [id];
+            }
+            const wordsStr = wordsArry.join(' ');
+            expect(label.innerHTML).toBe(`Show ${wordsStr}`);
+
+        })
+        
+    })
+    
+})
+
+
+describe(`Toggle buttons work as expected`, () => {
+
+    test(`Resume content visibility matches corresponding toggle button status on load`, () => {
+
+        const {getByTestId} = render(
+            <Provider store={store}><VisibilityChanger/></Provider>
+        )
+        const resume = render(<Provider store={store}><Resume/></Provider>);
+       
+        // note: "showSkillLevel" toggle button not being tested yet
+        const dataTestIDs = ["Email", "Phone", "Github", "LinkedIn", "Website", "Address", "Certification", "Education", "Experience", "Projects", "TechSkills"];
+        
+        dataTestIDs.map(id => {
+            // if toggle button status is on by default, corresponding resume content should also be visible
+            if(getByTestId(`show${id}`).children[0].classList.contains("checked")) {
+                expect(getByTestId(id)).toBeInTheDocument();
+            } else {
+                // if toggle button status is off by default, corresponding resume content should not be visible
+                expect(()=> {
+                    getByTestId(id) 
+                }).toThrow();
+            }
+        })
+            
+    })
+
+    test(`Toggle button click updates toggle button status and corresponding resume content visibility`, () => {
+
+        const {getByTestId} = render(
+            <Provider store={store}><VisibilityChanger/></Provider>
+        )
+        const resume = render(<Provider store={store}><Resume/></Provider>);
+
+        // note: "showSkillLevel" toggle button not being tested yet
+        const dataTestIDs = ["Email", "Phone", "Github", "LinkedIn", "Website", "Address", "Certification", "Education", "Experience", "Projects", "TechSkills"];
+        
+        dataTestIDs.map(id => {
+            // toggling off should update toggle button status from on to off, and corresponding resume content should disappear
+            if(getByTestId(`show${id}`).children[0].classList.contains("checked")) {
+                fireEvent.click(getByTestId(`show${id}`));
+                expect(getByTestId(`show${id}`).children[0].classList).not.toContain("checked");
+                expect(()=> {
+                    getByTestId(id) 
+                }).toThrow();
+            } else {
+                // toggling on should update toggle button status from off to on, and corresponding resume content should appear
+                fireEvent.click(getByTestId(`show${id}`));
+                expect(getByTestId(`show${id}`).children[0].classList).toContain("checked");
+                expect(getByTestId(id)).toBeInTheDocument();
+            }
+        })
+            
+    })
+
+
+})
+
```

Comment:
Please clear all of the `eslint` problems.

Reply:
Should be all set! Thanks